hvplot/converter.py

```diff
@@ -229,7 +232,8 @@ class HoloViewsConverter(object):
                      'min_height', 'min_width', 'frame_height', 'frame_width',
                      'aspect', 'data_aspect']
 
-    _style_options = ['color', 'alpha', 'colormap', 'fontsize', 'c', 'cmap']
+    _style_options = ['color', 'alpha', 'colormap', 'fontsize', 'c', 'cmap',
+                      'fontscale']
```

Comment:
Note it's not a plot_option, I realize it's misnamed but it should be in `_axis_options`.

Reply:
I have some questions still: you mention plot_option in your note, but do you mean it shouldn't be in _style_options, but instead in _axis_options?

Reply:
Sorry, yes, that's what I meant, edited.